Sming/SmingCore/Network/WebsocketClient.cpp

```diff
@@ -99,10 +100,11 @@ void WebsocketClient::onError(err_t err)
 
 bool WebsocketClient::_verifyKey(char* buf, int size)
 {
-	String dd = String(buf);
-	uint8_t s = dd.indexOf("Sec-WebSocket-Accept: ");
+	String keyHeader("Sec-WebSocket-Accept: ");
+	String dd(buf);
+	uint8_t s = dd.indexOfIgnoreCase(keyHeader);
```

Comment:
This is why in the first version I prefered to work directly with char* and not involve string copy constructor. We are playing with 1.5KB of heap alloc which you know it's not always available. I think I will revert to first solution of working directly on the buffer.